client/src/components/RepositoryItem/index.jsx

```diff
@@ -125,13 +125,19 @@ class RepositoryItem extends React.Component {
 
   render() {
     const {
-      repo: { name, isStar, isPublic },
+      repo: { name, isStar, isPublic, defaultBranch },
       username,
-      type
+      type,
     } = this.props;
     const { updatedAt, isEmpty, commitsPerYear } = this.state;
     const starsCount = Number(this.props.repo.starsCount);
     const repoType = isPublic ? '' : 'Private';
+    const language = defaultBranch
+    && defaultBranch.languageStats
+    && defaultBranch.languageStats.length
+    && defaultBranch.languageStats[0].language
+      ? defaultBranch.languageStats[0].language
+      : null;
```

Comment:
Theoretically this expression can be simplified into more readable: 
(e.g. const hasLanguageStats = defaultBranch && defaultBranch.languageStats; )

Reply:
It can be simplified indeed, but not further than `defaultBranch && defaultBranch.languageStats && defaultBranch.languageStats.length`, because an empty array returns true

Reply:
Oh, sorry, my bad, didn't notice that you suggested to move logical check into another variable. I'll do it, thanks.

Reply:
Agree, it's a matter of approach actually :)